vignettes/simulist.Rmd

```diff
@@ -0,0 +1,208 @@
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+This is an introductory vignette to the {simulist} R package. {simulist} simulates epidemiological data, either a line list, to provide a row-by-row account of cases in an infectious disease outbreak, or a contacts table, to provide a history of which individuals were in contact with which others.
+
+The main function in the {simulist} package is `sim_linelist()`. This functions takes in arguments that control the dynamics of the outbreak, such as the serial interval, and outputs a line list table (`<data.frame>`) with case information for each infected individual.
+
+For this introduction we will simulate a line list for a COVID-19 (SARS-CoV-2) outbreak. This will require two R packages: {simulist}, to produce the line list, and {epiparameter} to provide epidemiological parameters, such as onset-to-death delays.
+
+```{r setup}
+library(simulist)
+library(epiparameter)
+```
+
+First we load in some data that is required for the line list simulation. Data on epidemiological parameters and distributions is read from the {epiparameter} R package.
+
+```{r read-epidist}
+# get serial interval from {epiparameter} database
+
+serial_interval <- epidist(
+  disease = "COVID-19",
+  epi_dist = "serial interval",
+  prob_distribution = "gamma",
+  prob_distribution_params = c(shape = 1, scale = 1)
+)
+
+# get onset to hospital admission from {epiparameter} database
+onset_to_hosp <- epidist_db(
+  disease = "COVID-19",
+  epi_dist = "onset to hospitalisation",
+  single_epidist = TRUE
+)
+
+# get onset to death from {epiparameter} database
+onset_to_death <- epidist_db(
+  disease = "COVID-19",
+  epi_dist = "onset to death",
+  single_epidist = TRUE
+)
+```
+
+The reproduction number (`R`) is the first argument in `sim_linelist()` and with the serial interval will control the growth rate of the simulated epidemic. Here we set it as 1.1. The minimum requirements to simulate a line list are the reproduction number (`R`), the serial interval, onset-to-hospitalisation delay and onset-to-death delay.
+
+```{r, sim-linelist}
+linelist <- sim_linelist(
+  R = 1.1,
+  serial_interval = serial_interval,
+  onset_to_hosp = onset_to_hosp,
+  onset_to_death = onset_to_death
+)
+head(linelist)
+```
+
+## Case type 
+
+In the midst of an infectious disease outbreak it may not be possible to confirm every case. A confirmed case is usually done via a diagnostic test which may or may not need to be carried out in the laboratory. There are several reasons why a case may not be confirmed, one example is limited testing capacity, especially in fast growing epidemics. The other categories for cases are: probable and suspected. Probable cases are those that show clinical evidence for the disease but have not, or cannot, be confirmed by a diagnostic test. Suspected cases are those that are possibly infected but do not show clear clinical or epidemiological evidence, nor has a diagnostic test been performed.
```

Comment:
```suggestion
During an infectious disease outbreak it may not be possible to confirm every infection as a case. A confirmed case is typically defined via a diagnostic test. There are several reasons why a case may not be confirmed, including limited testing capacity and mild or non-specific early symptoms, especially in fast growing epidemics. We therefore include two other categories for cases: probable and suspected. For example, probable cases may those that show clinical evidence for the disease but have not, or cannot, be confirmed by a diagnostic test. Suspected cases are those that are possibly infected but do not show clear clinical or epidemiological evidence, nor has a diagnostic test been performed. Hence the distribution of suspected/probable/confirmed will depend on the pathogen characteristics, outreak-specific definitions, and resources available.
```

Reply:
Does the specific definition of probable/suspected/confirmed matter for the specific use case here? Often different organisations will define these slightly differently for each outbreak, so don't want users to get too distracted by the specifics if these are just 3 probabliistic defintions in the model.

Reply:
Thanks I will bear this in mind. I think the definitions given here are general enough to not contradict other organisations, or confuse users. If it becomes apparent from user feedback this is the case, we can reword or remove this section.

Reply:
Updated in 91286578306f249a66ce386bdc50d7ac1e71f280.